crates/pallet-domains/src/lib.rs

```diff
@@ -849,6 +849,10 @@ mod pallet {
             let extrinsics_root = opaque_bundle.extrinsics_root();
             let operator_id = opaque_bundle.operator_id();
             let receipt = opaque_bundle.into_receipt();
+            // `epoch_transitted` can't be `mut` with `runtime-benchmarks` otherwise clippy
+            // will complain, so simply skip the clippy rule
+            #[allow(unused_mut)]
```

Comment:
Can you do suppression conditionally compiled such that it is self-documenting and you don't need a comment on it?